packages/core/src/shorthands/types.test.ts

```diff
@@ -0,0 +1,25 @@
+import { type GriffelStyle } from '@griffel/style-types';
+import { borderBottom } from './borderBottom';
+
+describe('types tests', () => {
+  it('borderBottom', () => {
+    type Options = {
+      width?: GriffelStyle['borderBottomWidth'];
+      style?: GriffelStyle['borderBottomStyle'];
+      color?: GriffelStyle['borderBottomColor'];
+    };
+    const options: Options = {
+      width: '2px',
+      style: 'solid',
+      color: 'red',
+    };
+
+    const { width = '1px', style = 'solid', color = 'red' } = options;
+
+    expect(borderBottom(width, style, color)).toEqual({
```

Comment:
wondering if we should introduce test for all possible cases that can occur in order to mitigate future breaking changes on csstype level ?

Reply:
The test for other cases already exists, https://github.com/microsoft/griffel/blob/main/e2e/typescript/src/assets/fixture.ts.